pmg/user_management.py

```diff
@@ -64,6 +64,33 @@ def user_committee_alert(committee_id):
 
     return redirect(request.values.get('next', '/'))
 
+@app.route('/user/follow/committee/<int:committee_id>', methods=['POST'])
+def user_follow_committee(committee_id):
+    if current_user.is_authenticated() and request.method == 'POST':
+        u = current_user.follow_committee(Committee.query.get(committee_id))
```

Comment:
This code no longer works, because `follow_committee` doesn't return anything. SQLAlchemy knows how to track changed objects, you can simply do this:

```python
current_user.follow_committee(Committee.query.get(committee_id))
db.session.commit()
```